lib/feedsme/index.js

```diff
@@ -316,12 +362,15 @@ Feedsme.prototype._triggerOneDep = async function _triggerOneDep({ publish, prom
  * @returns {Object} strategy and whether we should trigger
  * @api private
  */
-Feedsme.prototype._triggerStrategy = function _triggerStrategy({ releaseLine, rootPkg, pkg, env }) {
+Feedsme.prototype._triggerStrategy = function _triggerStrategy({ registryRootPkg, releaseLine, rootPkg, pkg, env }) {
   if (env !== 'dev' && releaseLine) return { strategy: 'release', trigger: true };
   if (!releaseLine) return { strategy: 'legacy', trigger: true };
 
   const { name, version: rootVersion } = rootPkg;
   const { dependencies } = pkg;
+  const rootPkgVersions = Object.keys(registryRootPkg.versions || {})
```

Comment:
@msluther yea this actually should never happen. I can remove this or be even more defensive. Im not sure if registry.get actually errors on a 404. if it does this would 100% never happen, if not then being more defensive cant hurt.